### PR TITLE
Fixed pytorch_lightning warning

### PR DESCRIPTION
**Pull request by StephanAkkerman**
Renamed "val_Loss" to "val_loss" to prevent to following error:  WARNING pytorch_lightning.py:142 -- Metric val_Loss does not exist in `trainer.callback_metrics.

Fixes no issue found for this.

### Summary

This fixes the Pytorch Lightning warning users get when running the code for Ray Tune. It might confuse new users of Darts when running the example, wondering if they did something wrong.

### CODE REVIEW

**Review by dennisbader — APPROVED** [review]
Thanks @StephanAkkerman , looks good to me 🚀
